### PR TITLE
Update deleting-your-user-account.md

### PR DESCRIPTION
**Pull request by aleksandrpidgurskij**
### Why:

**Closes [issue link]**

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by aleksandrpidgurskij — COMMENTED** [review]
Hi!!

**Review by lp9921 — COMMENTED** [review]
aleksandrpidgurskij:patch-2

**Comment by lp9921** [comment]
aleksandrpidgurskij:patch-2

File: content/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+9
```
Comment (lp9921) [inline content/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account.md:1]: aleksandrpidgurskij:patch-2

Comment (lp9921) [reply content/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account.md:1]: content/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account.md

**Review by lp9921 — APPROVED** [review]
content/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account.md

**Comment by lp9921** [comment]
aleksandrpidgurskij:patch-2

**Pull request closed without merging**